sourcemap/mapper_test.go

```diff
@@ -32,17 +30,17 @@ import (
 
 func TestMapNilConsumer(t *testing.T) {
 	// no sourcemapConsumer
-	_, _, _, _, _, _, _, ok := Map(nil, 0, 0)
+	_, _, _, _, _, _, _, ok := Map(nil, 0, 0, 0)
 	assert.False(t, ok)
 }
 
 func TestMapNoMatch(t *testing.T) {
 	m, err := sourcemap.Parse("", []byte(test.ValidSourcemap))
-	require.NoError(t, err)
+	assert.NoError(t, err)
```

Comment:
why changing to `assert`? Running the tests if the parsing failed doesn't make much sense and would probably just lead to confusing errors (hiding the original issue). Similar for other changes below.

Reply:
ah sorry, just my OCD. I'll change them back